python/ray/tune/utils/util.py

```diff
@@ -471,18 +471,16 @@ def wait_for_gpu(gpu_id=None, gpu_memory_limit=0.1, retry=20):
         gpu_id (Optional[str]): GPU id to check. Must be found
             within GPUtil.getGPUs(). If none, resorts to
             the first item returned from `ray.get_gpu_ids()`.
-        gpu_memory_limit (float): If memory usage is below
+        gpu_memory_limit (float): If fractional memory usage is below
```

Comment:
I think the original docstring was correct here. We're comparing absolute memory usage (`memoryUsed`) vs. the absolute amount we need to have available (`gpu_memory_limit`). The fact that the default value here is `0.1` might be confusing. Shouldn't we just require passing a GPU limit here? And maybe state that this is memory usage in bytes (if this is in bytes, which I think it is?)

Reply:
Can you keep it as a fraction and compare against `memoryUtil` instead of `memoryUsed`?

https://github.com/anderskm/gputil/blob/master/GPUtil/GPUtil.py#L50

This is more intuitive for us, particularly since in most cases you just want to say "wait until all the gpu memory is free" or `wait_for_gpu(gpu, 1.0)`.

Reply:
done!

Reply:
so now, the usage is:

`wait_for_gpu(target_util=0.1)` or `wait_for_gpu(target_util=0)` (for full blocking)